openslides_backend/action/action.py

```diff
@@ -673,6 +675,19 @@ def get_on_failure(self, action_data: ActionData) -> Callable[[], None]:
         after an error appeared in an action.
         """
 
+    def get_translate_function(self) -> Callable[[str], str]:
+        def null_translate(msg: str) -> str:
+            return msg
+
+        if not hasattr(self, "language") or not self.language:
+            return null_translate
+        t = gettext.translation(
+            "messages", "/app/openslides_backend/locale", [self.language]
+        )
+        if not t:
+            return null_translate
+        return t.gettext
```

Comment:
Please extract this into a singleton/static class where the current language is always saved and the translate function can be accessed. I don't like polluting the action class any more

Reply:
We need a translation function per request. So I need some kind of factory, not a singleton.

Reply:
Yes, but a factory instance will have to be passed through, which is not what we want. Since we only handle one request at a time, a singleton should suffice.

Reply:
Okay, 2 requests should not be a problem, introduce a singleton Translator.